Cargo.toml

```
@@ -14,6 +14,7 @@ bounded-integer-macro = { path = "./macro", version = "=0.5.0", optional = true
 
 arbitrary1 = { package = "arbitrary", version = "1.0.2", optional = true }
 bytemuck1 = { package = "bytemuck", version = "1.7.2", optional = true }
+num-traits = { package = "num-traits", version = "0.2.14", default-features = false, optional = true }
```

Can you change the feature name to `num-traits02`? This will avoid our semver major number being tied to `num-traits`.

```suggestion
num-traits02 = { package = "num-traits", version = "0.2.14", default-features = false, optional = true }
```

---

Sure. I couldn't find a note on `num-traits`'s versioning policy but 0.1 -> 0.2 was a breaking change, so I'm happy with relying on the minor version here.